### PR TITLE
adding sibilant and rhythm server stuff to the basic demo

### PR DESCRIPTION
**Pull request by dcalacci**

### CODE REVIEW

**Comment by sarahkemi** [comment]
Figured out why a meetings collection wasn't being made. I didn't format the participants array the way google hangouts does, so it was breaking this (in meetingJoinedEvent.js):
`app.service('meetings').get(data.meeting)
 .then((meeting) => {
                 return app.service('meetings').patch(meeting._id, {
                   participants: participantIds
                 }).then((meeting) => {
                   return meeting
                 })
})`
and it would just make a new meeting every time.

However, it's temporarily using the room name as a meeting id, which might suck in the future....

**Comment by dcalacci** [comment]
Aha!
Yes, we'll have to generate random meeting IDs for rooms in the future

**Comment by sarahkemi** [comment]
Okay, so now I'm struggling to recreate the possible race condition! It seems that it occurs more on the heroku deployment than my local instance. It also is more likely to occur after restarting the rtc server. 

Now that I've managed to recreate a few times, it looks consent is breaking it. Even though the promises are chained, sometimes the consent isn't given and audio isn't processed.

File: client/rtcClient.js [hunk 1]
```diff
@@ -63,21 +63,15 @@ function loginSuccess () {
...
       meetings: $scope.roomName,
       meeting: $scope.roomName,
-      meetingUrl: location.href
-    })
-  }).catch(function (err) {
-    console.log('ERROR:', err)
-  }).then(function (result) {
-    console.log('meeting result:', result)
-    return app.service('participants').patch(easyrtc.myEasyrtcid, {
+      meetingUrl: location.href,
```
Comment (dcalacci) [inline client/rtcClient.js:66]: ?

Comment (sarahkemi) [inline client/rtcClient.js:66]: Hm?

Comment (dcalacci) [inline client/rtcClient.js:66]: sorry, not being specific -- why is there a meetingUrl being patched to a participants object? IIRC meetingUrl is only something that exists in a meeting object, shouldn't this break things?

Comment (sarahkemi) [inline client/rtcClient.js:66]: Oh, it's not being patched to the participants object. All that code was removed!

**Pull request merged**